src/quo2/components/list_items/account/view.cljs

```diff
@@ -30,6 +38,9 @@
                    colors/white-opa-40
                    (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))}]])]
     [text/text {:size :paragraph-2}
+     (for [network (:networks account-props)]
+       ^{:key (str network)}
+       [network-view network])
```

Comment:
It would be tiring work to change network names, colors, and font across the codebase If something happens like this in future.

We need to abstract this rendering of the network list and the (text) view into a component (and use it in places where it is needed) for better maintainability. WDYT?

Reply:
Created an issue https://github.com/status-im/status-mobile/issues/17628